Nodejs/Product/TestAdapter/TestFrameworks/NodejsTestInfo.cs

```diff
@@ -10,8 +10,6 @@ public sealed class NodejsTestInfo
     {
         public NodejsTestInfo(string testPath, string suite, string testName, string testFramework, int line, int column, string projectRootDir)
         {
-            Debug.Assert(testPath.EndsWith(".js", StringComparison.OrdinalIgnoreCase) || testPath.EndsWith(".jsx", StringComparison.OrdinalIgnoreCase));
-
             var testFileRelative = CommonUtils.GetRelativeFilePath(projectRootDir, testPath);
```

Comment:
I believe removing this assertion makes sense, as this information should come from the test framework. In the case of angular the default paths will have a `.ts` extension. This is true for other test frameworks as well. E.g. jest might return tests with `.ts` or `.tsx` as extension. 

Those are the only extensions I have seen but I wouldn't be surprised is somebody comes with something else. (looking at `.mjs`)

Let me know if you feel otherwise.